classes/publication/enums/JavStage.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file publication/enums/JavStage.php
+ *
+ * Copyright (c) 2023-2024 Simon Fraser University
+ * Copyright (c) 2023-2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class JavStage
+ *
+ * @brief Enumeration for JAV versioning stages
+ */
+
+namespace PKP\publication\enums;
+
+enum JavStage: string
+{
+    case AUTHOR_ORIGINAL = 'AO';
```

Comment:
While these are fine to use in the codebase, we should be careful not to present them straight to the front end without having them map to translatable locale keys.